zerver/migrations/0387_customprofilefield_display_in_profile_popover.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.12 on 2022-03-28 21:18
```

Comment:
Duplicate migration number `0387`, this should be `388`.

Reply:
I think I need to rebase here.